### PR TITLE
Update `ru_ru.json`

### PR DESCRIPTION
**Pull request by mpustovoi**
Added missing lines.

I'm unable to cover all possible combinations with accessibility elements due to the presence of three genera in the Russian language. By default, I used the masculine gender (but maybe I should have used the neuter, although it would still look wrong).
I guess some prefixes refer only to specific blocks/entities (like sheep), but I don't want to break the consistency.

### CODE REVIEW

File: src/main/resources/assets/jade/lang/ru_ru.json [hunk 1]
```diff
@@ -281,6 +321,81 @@
...
+	"jade.access.creeper.powered": "Заряженный %s",
+	"jade.access.zombie_villager.curing": "Исцеляющийся %s",
+	"jade.access.goat.hornless": "Безрогий %s",
+	"jade.access.wither.powered": "Запитанный %s",
+	"jade.access.entity.sheared": "Стриженный %s",
+	"jade.access.entity.saddled": "Осёдланный %s",
+	"jade.access.entity.baby": "Детёныш %s",
+	"jade.access.entity.angry": "Сердитый %s",
+	"jade.access.bee.nectar": "Нектарный %s",
+	"jade.access.slime.size": "{1,выбор,1#Малый|2≤Средний|4≤Большой|8≤Огромный} {0}",
```
Comment (Snownee) [inline src/main/resources/assets/jade/lang/ru_ru.json]: choice should not be translated

**Review by Snownee — CHANGES_REQUESTED** [review]

**Review by Snownee — APPROVED** [review]

**Comment by Snownee** [comment]
Hi, I just added grammatical gender support to it. (https://github.com/Snownee/Jade/commit/7095be3f689e2353ba07a0886b81c5aac43ba31b) The explanation can be found in the README.txt. You can use it, or just help me change the “%s" variables to "{0}" in your translation. Thank you!

**Comment by mpustovoi** [comment]
Done.
Although it still seems to me that I didn't quite understand the usage correctly. Here I think I need add variants to adjectives. However, I won't have time to do this anytime soon. If I see an example later, I think I can finish it. I guess it will be already next year.

File: src/main/resources/assets/jade/lang/ru_ru.json [hunk 2]
```diff
@@ -281,6 +321,81 @@
...
 	"config.jade.plugin_minecraft.waxed": "Вощёное состояние",
 	"config.jade.plugin_minecraft.waxed_desc": "Показывать вощёное состояние блока, например, вощёная табличка.",
+	"jade.waxed": "Waxed %s",
+	"config.jade.accessibility_plugin": "Плагин специальных возможностей",
+	"config.jade.accessibility_plugin_desc": "Включите специальные возможности для отображения и передачи более подробной информации. После этого вы сможете управлять более подробной информацией в настройках плагина.",
+	"config.jade.plugin_jade_access": "Специальные возможности",
+	"config.jade.plugin_jade_access.sign": "Содержимое таблички",
+	"jade.access.sign.front": "Спереди:",
+	"jade.access.sign.back": "Сзади:",
+	"jade.access.sign.line1": "Первая строка: {0}",
```
Comment (Snownee) [inline src/main/resources/assets/jade/lang/ru_ru.json]: variables after ": " should not be changed

**Review by Snownee — CHANGES_REQUESTED** [review]

**Pull request merged**